troposphere/elasticsearch.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2012-2015, Mark Peek <mark@peek.org>
+# All rights reserved.
+#
+# See LICENSE file for full license.
+
+from . import AWSHelperFn, AWSProperty, AWSObject
+from .validators import boolean, integer, integer_range, positive_integer
+
+VALID_VOLUME_TYPES = ('standard', 'gp2', 'io1')
+
+try:
+    from awacs.aws import Policy
+    policytypes = (dict, Policy)
+except ImportError:
+    policytypes = dict,
+
+
+def validate_volume_type(volume_type):
+    """Validate VolumeType for ElasticsearchDomain"""
+    if isinstance(volume_type, AWSHelperFn):
```

Comment:
You shouldn't need this, due to this:

https://github.com/cloudtools/troposphere/blob/master/troposphere/__init__.py#L99